inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/querybuilder/SPARQLQueryBuilderTest.java

```diff
@@ -167,19 +177,100 @@ public void setup()
         // https://dbpedia.org/sparql
         dbpedia = new SPARQLRepository("https://dbpedia.org/sparql");
         dbpedia.initialize();
-        
+
+        // https://linkeddata1.calcul.u-psud.fr/sparql
+        yago = new SPARQLRepository("https://linkeddata1.calcul.u-psud.fr/sparql");
+        yago.initialize();
+
         // http://nlp.dainst.org:8888/sparql
         hucit = new SPARQLRepository("http://nlp.dainst.org:8888/sparql");
         hucit.initialize();
+        
+        // http://collection.britishmuseum.org/sparql
+        britishMuseum = new SPARQLRepository("http://collection.britishmuseum.org/sparql");
+        britishMuseum.initialize();
     }
     
+    @Test
+    public void thatExistsQueryReturnsTrue() throws Exception
```

Comment:
This misses the actual data you use, so I do not know why it should be the case. Same with the test below.
thatExistsQueryReturnsTrueIfEntryExists

Reply:
Renamed and added JavaDoc.